src/test/resources/crashes/minecraft/modded-level-file-parsed.txt

```diff
@@ -1,8 +1,9 @@
   "minecraftVersion" : "1.16.5",
   "isClient" : false,
-  "deobf" : null,
```

Comment:
This change comes from the fact that the crash report is now not considered modded anymore and is therefore deobfuscated.